web/src/utilities/utilities.js

```diff
@@ -700,8 +700,11 @@ export const Utilities = {
       window.localStorage.removeItem("session_roles");
     }
 
-    if (window.location.pathname !== "/secure-console") {
-      window.location = "/secure-console";
+    const redirectPath = options.snapshotRestore
+      ? "/restore-complete"
+      : "/secure-console";
+    if (window.location.pathname !== redirectPath) {
+      window.location = redirectPath;
```

Comment:
eventually we'll refactor this out and use react router since this causes a hard refresh